standard/table.lua

```diff
@@ -67,16 +67,10 @@ end
 ---@param tbl table?
 ---@return boolean
 function Table.isEmpty(tbl)
-	if tbl == nil then
+	if not tbl then
 		return true
 	end
-	-- luacheck: push ignore
-	--it is intended that the loop is executed at most once
-	for _, _ in pairs(tbl) do
-		return false
-	end
-	-- luacheck: pop
-	return true
+	return next(tbl) and true or false
```

Comment:
logic is the wrong way around
(this returns `true` for non empty tables and `false` for empty ones)

Reply:
changed `return next(tbl) and true or false` to `return not next(tbl) and true or false`